worker/mongoupgrader/manifold.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2017 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package mongoupgrader
```

Comment:
All of this mongo upgrade stuff
- the worker
- the facade apis
- the machine doc attribute

... none of it is used. There's an upgrade-mongo plugin which is the only thing which calls the facade api, and we are dropping the plugin as it is not used. It was only for back in the 1.26 days when we were going to upgrade mongo. 

As a first step to getting of this stuff. let's drop the worker from this PR. A followup can remove the facades and other bits.

Reply:
ok, added a commit to drop the worker and manifold